### PR TITLE
Link to data generation article from incremental property testing.

### PR DESCRIPTION
**Pull request by markrwilliams**
The end of "Evolving toward property-based testing with Hypothesis"
implies that a repositories strategy could refine the property under
test, but does not explain how to write it.  Readers who are linked
directly to this article won't know that "Generating the right data",
the next article in the recommended reading list, explains how to
write a strategy that instantiates arbitrary Python classes.

Explicitly state that Hypothesis can generate classes like Repository
and link to "Generating the right data" so interested readers can
learn how generate non-primitive data types.

### CODE REVIEW

File: _posts/2016-06-05-incremental-property-based-testing.md [hunk 1]
```diff
@@ -249,9 +249,12 @@ Going forward
...
 We stopped there, but we need not have. Just as the test should have held for
 any branch, it should also hold for any repository. We were just creating an
-empty repository because it was convenient for us.
+empty repository because it fit our focus here. [Hypothesis' data generation
+toolkit can construct instances of non-primitive data types](
+{% post_url 2016-05-11-generating-the-right-data %}) like our `Repository`
+class.
 
-If we were to continue, the test would have looked something like this:
+The next step would be to write a `repositories` strategy for our test:
```
Comment (DRMacIver) [inline _posts/2016-06-05-incremental-property-based-testing.md]: I find the follow-on doesn't quite work here - the way this is phrased suggests it's about to show how to do that, but instead it just shows an instance of using it. Maybe rephrase?

**Review by DRMacIver — CHANGES_REQUESTED** [review]
Good idea, yes. Thank you!

This is @jml's article originally, so I think it should be up to him to approve, but I've one comment about phrasing inline.

**Comment by jml** [comment]
Agree in principle. I trust @DRMacIver's judgement on the details.
